mix.exs

```diff
@@ -126,39 +126,67 @@ defmodule Livebook.MixProject do
   ## Releases
 
   defp releases do
+    macos_notarization = macos_notarization()
+
     [
       livebook: [
         include_executables_for: [:unix],
         include_erts: false,
         rel_templates_path: "rel/server",
         steps: [:assemble, &remove_cookie/1]
       ],
-      mac_app: [
-        include_executables_for: [:unix],
-        include_erts: false,
-        rel_templates_path: "rel/app",
-        steps: [:assemble, &remove_cookie/1, &standalone_erlang_elixir/1, &build_mac_app/1]
-      ],
-      mac_app_dmg: [
-        include_executables_for: [:unix],
-        include_erts: false,
-        rel_templates_path: "rel/app",
-        steps: [:assemble, &remove_cookie/1, &standalone_erlang_elixir/1, &build_mac_app_dmg/1]
-      ],
-      windows_installer: [
-        include_executables_for: [:windows],
+      app: [
         include_erts: false,
         rel_templates_path: "rel/app",
         steps: [
           :assemble,
           &remove_cookie/1,
           &standalone_erlang_elixir/1,
-          &build_windows_installer/1
+          &AppBuilder.bundle/1
+        ],
+        app: [
+          name: "Livebook",
+          icon_path: [
+            macos: "rel/app/icon-macos.png",
+            windows: "rel/app/icon.ico"
+          ],
+          url_schemes: ["livebook"],
+          document_types: [
+            %{
+              name: "LiveMarkdown",
+              extensions: ["livemd"],
+              icon_path: [
+                macos: "rel/app/icon.png",
+                windows: "rel/app/icon.ico"
+              ],
+              macos_role: "Editor"
+            }
+          ],
+          additional_paths: [
+            "rel/erts-#{:erlang.system_info(:version)}/bin",
+            "rel/vendor/elixir/bin"
+          ],
+          server: LivebookApp,
+          macos_is_agent_app: true,
```

Comment:
I would probably have this be not a boolean: `macos_app_type: :regular | :agent`. :)

Reply:
sounds good!